src/libraries/System.Net.Http/src/System/Net/Http/Headers/EntityTagHeaderValue.cs

```diff
@@ -8,9 +8,7 @@ namespace System.Net.Http.Headers
 {
     public class EntityTagHeaderValue : ICloneable
     {
-        private static EntityTagHeaderValue? s_any;
-
-        private string _tag = null!;
+        private string _tag;
```

Comment:
These can be readonly now, right?

Reply:
(These == _tag and _isWeak)

Reply:
They're currently written in GetEntityTagLength.  I'll leave that for someone to explore another time :)